File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;


int solution(vector<string> strArr) {
    vector<int> arr(99999,0);
    
    for (int i = 0; i < strArr.size(); i++) arr[strArr[i].size()]++;
    
    return *max_element(arr.begin(),arr.end());
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        int[] arr = new int[31];
        
        for (String str : strArr) {
            arr[str.length()]++;
        }
        
        for (int i : arr) {
            answer = Math.max(answer, i);
        }
        
        return answer;
    }
}